backend/script/format

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+poetry run black app
```

Comment:
I've added `isort` to my format scripts too. Run it before black because black will "correct" some of its spacing. That also means that you can't run `isort --check` in your test script.